frontend/src/views/LevelView.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="Container">
+    <img src="@/assets/images/logo-and-lettering.svg" alt="Saci Logo" />
+    <h1>{{ newLevelTitle }}</h1>
+    <h2>Basic Information</h2>
+    <div class="NameContainer">
+      <h5>Name</h5>
+      <ElInput placeholder="Level Name" v-model="newLevel.name" />
+    </div>
+    <div class="FlexContainer">
+      <div class="MaxAndMinInputs">
+        <h5>Coeficient Range</h5>
+        <span>
+          <ElInput placeholder="Min Range" v-model="newLevel.minCoeficient"></ElInput>
+          <h4>to</h4>
+          <ElInput placeholder="Max Range" v-model="newLevel.maxCoeficient"></ElInput>
+        </span>
+      </div>
+      <div class="LinkSide">
```

Comment:
```suggestion
      <div class="LinkField">
```

Clear class names, please